types/google-maps-web/tslint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-qualifier": false,
+        "strict-export-declare-modifiers": false,
+        "use-default-type-parameter": false,
+        "no-mergeable-namespace": false,
+        "ban-types": false,
+        "npm-naming": false
```

Comment:
If you remove this exception, you'll have to start the header with "Type definition for the non-npm package ...", but I think it should otherwise work, as long as npm doesn't have a package named `google-maps-web`. Leaving the rule on lets us know in case a conflict arises in the future.

Reply:
fixed npm-naming rule